services/reddit/subreddit-subscribers.tester.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const { isMetric } = require('../test-validators')
+const { ServiceTester } = require('../tester')
```

Comment:
We can use the standard `createServiceTester` helper here, see the [tests tutorial](https://github.com/badges/shields/blob/master/doc/service-tests.md#1-boilerplate) for more info